public/admin-account.php

```diff
@@ -4,7 +4,11 @@
   require_once("../utils/filter.php");
   require_once("../utils/movies.php");
   require_once("../utils/functions.php");
-  checkLoggedIn();
+  checkSessionLoggedIn();
+
+  if(!checkDatabaseLoggedIn($_SESSION['email'])){
+    header('Location: ./login.php');
+  }

```

Comment:
Voor deze en alle andere checks, hij is nice, ik zou hem alleen wel op het id pakken van de gebruiker. Dit is ook een unieke identifier van een account en dat scheelt weer een extra $_SESSION variabele